extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -40,6 +44,26 @@ describes.realWin('amp-pinterest', {
     });
   };
 
+  function getEmbedPin(pinID, pinAlt, mockResponse) {
+    const div = document.createElement('div');
+    env.win.document.body.appendChild(div);
+
+    env.fetchMock.getOnce(widgetURL(pinID), {'body': mockResponse});
+
+    const pinURL = 'https://www.pinterest.com/pin/';
+    const pin = env.win.document.createElement('amp-pinterest');
+    pin.setAttribute('data-do', 'embedPin');
+    pin.setAttribute('data-url', pinURL + pinID);
+    if (pinAlt) {
```

Comment:
this was needed since `setAttribute` stringifies so it was setting alt to `null` string in two of the cases